src/components/networks/networkOverviewTab.jsx

```diff
@@ -53,7 +56,47 @@ const DHCPHost = (host, index, family, idPrefix) => {
 
     const hostInfo = hostVals.join(", ");
 
-    return <ListItem key={index} id={id}> {hostInfo} </ListItem>;
+    const removeDHCPHostButton = (
+        <DeleteResourceButton objectId={`${id}-button`}
+            actionName={_("remove")}
+            variant="link"
+            isInline
+            showDialog={() => setDeleteDialogProps({
+                objectType: _("static host from DHCP"),
+                objectName: "",
+                actionDescription: _(`The static host entry for ${host.ip} will be removed:`),
+                objectDescription: [
+                    { name: _("IP"), value: <span className="ct-monospace">{host.ip}</span> },
+                    { name: _("MAC"), value: <span className="ct-monospace">{host.mac}</span> }
+                ],
+                actionName: _("Remove"),
+                onClose: () => setDeleteDialogProps(undefined),
+                deleteHandler: () => {
+                    return networkRemoveStaticHostEntries({
```

Comment:
Please drop the `{ return ... }` wrapper, and keep this functional (`() => networkRemoveStaticHostEntries({...`)